geo/src/algorithm/relate/geomgraph/intersection_matrix.rs

```diff
@@ -220,6 +220,11 @@ impl IntersectionMatrix {
             && self.0[CoordPos::Outside][CoordPos::Inside] == Dimensions::Empty
             && self.0[CoordPos::Outside][CoordPos::OnBoundary] == Dimensions::Empty
     }
+
+    /// Directly accesses this matrix
+    pub fn get(&self, lhs: CoordPos, rhs: CoordPos) -> Dimensions {
```

Comment:
Please add an example usage doc test — the argument type and positions seem a little tricky if you don't already understand what the are.